src/frontend/komponenter/Fagsak/Vedtak/VedtakBegrunnelserTabell/VedtaksBegrunnelsePanel.tsx

```diff
@@ -0,0 +1,199 @@
+                        <Hjelpetekst ref={element => settHjelpetekstRef(element)}>
+                            Dette er en hjelpetekst (tekst kommer senere)
+                        </Hjelpetekst>
+                    </HjelpetekstWrapper>
+                    <span className="typo-element">
+                        {periodeToString({
+                            fom: utbetalingsperiode.periodeFom,
+                            tom: slutterSenereEnnInneværendeMåned(utbetalingsperiode.periodeTom)
+                                ? ''
+                                : utbetalingsperiode.periodeTom,
+                        })}
+                    </span>
+                    <span className="typo-normal">Ordinær</span>
+                    <span className="typo-normal">{utbetalingsperiode.utbetaltPerMnd} kr</span>
+                </UtbetalingsperiodepanelTittel>
+            }
+        >
+            <UtbetalingsperiodepanelBody>
+                <div>
+                    <Element>Resultat</Element>
+
+                    {utbetalingsperiode.utbetalingsperiodeDetaljer.map(
+                        (detalj: IUtbetalingsperiodeDetalj) => (
+                            <UtbetalingsperiodeDetalj key={detalj.person.personIdent}>
+                                <Normaltekst
+                                    onMouseEnter={event =>
+                                        settFødselsnummerAnker({
+                                            fødselsnummer: detalj.person.personIdent,
+                                            element: event.currentTarget,
+                                        })
+                                    }
+                                    onMouseLeave={() => settFødselsnummerAnker(undefined)}
+                                >
+                                    {displayFødselsnummer(detalj.person.personIdent)}
+                                </Normaltekst>
+
+                                <Popover
+                                    ankerEl={
+                                        fødselsnummerAnker?.fødselsnummer ===
+                                        detalj.person.personIdent
+                                            ? fødselsnummerAnker.element
+                                            : undefined
+                                    }
+                                    orientering={PopoverOrientering.Under}
+                                    autoFokus={false}
+                                    tabIndex={-1}
+                                >
+                                    <StyledNormaltekst>{detalj.person.navn}</StyledNormaltekst>
+                                </Popover>
+
+                                <Normaltekst>{detalj.utbetaltPerMnd} kr</Normaltekst>
+                            </UtbetalingsperiodeDetalj>
+                        )
+                    )}
+                </div>
+                <div>
+                    <Element>Begrunnelse(r) i brev</Element>
```

Comment:
Er dette label'en til input? Isåfall bør vi heller bruke label prop til multiselect :)